tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/typecast/TypeCastExprTest.java

```diff
@@ -529,7 +529,7 @@ public void testMapToAny() {
     @Test(description = "Test casting a struct to another struct in a different package")
     public void testCastToStructInDifferentPkg() {
 //        CompileResult res = BCompileUtil.compile("test-src/expressions.typecast.foo", "expressions.typecast.foo");
```

Comment:
Shall we remove this line ?

Reply:
+1. Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/26802/commits/f6aa23950cf37c29926c3e48d1cc2492591fb017